app/src/signing/networkMessages/polkadot.js

```diff
@@ -158,20 +158,30 @@ export async function ClaimRewardsTx(senderAddress, {}, network, api) {
 // Vote
 export async function VoteTx(
```

Comment:
instead create a different Tx for seconding?

Reply:
I don't think this is needed. According to the parameters we see what type of vote we need. And there are only two types, so I think it is fine like this